.gitignore

```diff
@@ -9,3 +9,4 @@
 /spec/mercure.xml
 __pycache__
 *.pyc
+.idea/
```

Comment:
Can you remove this? It should be in a global gitignore file.

Reply:
Alright, will do